server/routes/journeys/update-plan/planned-intervention/schemas.ts

```diff
@@ -0,0 +1,29 @@
+import z from 'zod'
+import { Request, Response } from 'express'
+import { createSchema } from '../../../../middleware/validationMiddleware'
+import { getMaxCharsAndThresholdForAppend } from '../../../../utils/appendFieldUtils'
+
+const INTERVENTION_REQUIRED_MSG = 'Enter an update on the planned intervention'
+
+const TOO_LONG_ERROR_MSG = (isUpdate: boolean | undefined, maxLengthChars: number) =>
+  `${isUpdate ? 'Update to the planned intervention' : 'Planned intervention for the identified need'} must be ${maxLengthChars.toLocaleString()} characters or less`
```

Comment:
this schema is only used in Update journey, i don't think we need this conditional check?